civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1117,29 +1117,61 @@ function removeAdditionalCommas($view) {
   return $fieldsToClean;
 }
 
-function removeInactiveDepartments($results = array()) {
-  foreach ($results as $key => $value) {
-    $startDate = !empty($value->hrjc_role_civicrm_contact_start_date) ? $value->hrjc_role_civicrm_contact_start_date : NULL;
-    $endDate = !empty($value->hrjc_role_civicrm_contact_end_date) ? $value->hrjc_role_civicrm_contact_end_date : NULL;
+/**
+ * Sets the name to NULL for relationships starting in future or ending in past
```

Comment:
Originally it was just setting the manager name to NULL but this resulted in fields with extra commas like ", Contact B"

I saw the query returns multiple results and removing those rows with the inactive relationships got rid of the data I didn't want while preserving everything else.